Cart_Reader/Cart_Reader.ino

```diff
@@ -138,6 +155,7 @@ typedef enum COLOR_T {
 #define mode_GB_GBSmart_Game 20
 #define mode_WS 21
 #define mode_NGP 22
+#define mode_RTC 23

```

Comment:
This can be omitted, was setting up another select-able option for the user to set the time but scrapped the idea.